doc/tags.txt

```diff
@@ -1,3 +1,18 @@
+v3_10_9
+	Added custom JWT-authenticated log server example (new RPM glideinwms-logging) (Issue#398, PR#467)
+	Now using also Apptainer included in the HTCondor tar ball (Issue#364, PR#473)
+	Improved the docstrings and some code in the lib files and few others with the help of AI (PR#471, PR#472)
+	Bug fix: Fixed early truncation in log files configuration and inconsistent documentation (Issue#464, PR#462, PR#463)
+	Bug fix: Removed confusing tac broken pipe messages from the Glidein stderr (PR#465)
+	Bug fix: Fixed JWT logging credentials not transferred to the Glidein. This includes removal of DictFile.append() and use of add_environment() for JWT tokens (Issue#398, PR#467)
+	Bug fix: Fixed quotes in Glidein command line unpacking and replaced deprecated add_config_line commands (PR#468)
+	Bug fix: Allow anonymous SSL authentication for the dynamically generated client config (Issue#222, PR#470)
+	Bug fix: Checking also the apptainer binary in the SINGULARITY_BIN path, not only singularity (PR#473)
+	NOTE: The Glidein will always send SIGQUIT to HTCondor when it receives INT, TERM, QUIT signals. This speeds up the shutdown (PR#466)
+	NOTE: Renamed Glidein Logging functions to glog_... (glog_init, glog_setup, glog_write, glog_send - PR#467) The original functions (log_init_tokens, log_init, log_setup, log_write, send_logs_to_remote, ...) are no more available
```

Comment:
same comment as with the previous file, if applicable (given that this file and the previous one might be auto generated)